src/main/java/org/jabref/logic/shared/PostgreSQLProcessor.java

```diff
@@ -41,24 +44,37 @@ public void setUp() throws SQLException {
             VERSION_DB_STRUCT_DEFAULT = 0;
         }
 
-        connection.createStatement().executeUpdate("CREATE SCHEMA IF NOT EXISTS jabref");
-
-        connection.createStatement().executeUpdate(
-                "CREATE TABLE IF NOT EXISTS " + escape_Table("ENTRY") + " (" +
-                        "\"SHARED_ID\" SERIAL PRIMARY KEY, " +
-                        "\"TYPE\" VARCHAR, " +
-                        "\"VERSION\" INTEGER DEFAULT 1)");
-
-        connection.createStatement().executeUpdate(
-                "CREATE TABLE IF NOT EXISTS " + escape_Table("FIELD") + " (" +
-                        "\"ENTRY_SHARED_ID\" INTEGER REFERENCES " + escape_Table("ENTRY") + "(\"SHARED_ID\") ON DELETE CASCADE, " +
-                        "\"NAME\" VARCHAR, " +
-                        "\"VALUE\" TEXT)");
-
-        connection.createStatement().executeUpdate(
-                "CREATE TABLE IF NOT EXISTS " + escape_Table("METADATA") + " ("
-                        + "\"KEY\" VARCHAR,"
-                        + "\"VALUE\" TEXT)");
+        // TODO: Before a release, fix the names (and migrate data to the new names)
+        //       Think of using Flyway or Liquibase instead of manual migration
```

Comment:
I have experience with liquibase, we use it at work.they offer a gralde plugin https://contribute.liquibase.com/extensions-integrations/directory/integration-docs/gradle/

Reply:
https://forum.liquibase.org/t/adding-liquibase-to-an-existing-project/6076